pkg/config/parameters.go

```diff
@@ -59,7 +66,7 @@ func (g GatewayParameters) Validate() error {
 	}
 
 	if len(errorString) > 0 {
-		return fmt.Errorf("invalid Gateway parameters specified: %s", errorString)
+		return fmt.Errorf("invalid ConfiguredGateway parameters specified: %s", errorString)
```

Comment:
Should this and the related changes in `pkg/config/` be reverted? Maybe some other find/replaces that could be reviewed and reverted where it makes sense?

Reply:
bah my goland refactor tool doesn't always do the right thing. I'll swap those back.